<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitFileDialog</class>
 <widget class="QDialog" name="SplitFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete lines dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_SplitByLines">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="NumberOfLines_label">
        <property name="text">
         <string>Number of lines:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="NumberOfLines_lineEdit_1"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>132</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number of header lines:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="NumberOfHeaderLines_lineEdit_1"/>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>132</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_SplitByLargeFiles">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Number of header lines:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="NumberOfHeaderLines_lineEdit_2"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>132</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_SplitByColumns">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Number of columns:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="NumberOfColumns_lineEdit_3"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>105</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Number of metadata columns:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="NumberOfMetadataColumns_lineEdit_3"/>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>105</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_Options">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="SkipEmptyLines_checkBox">
        <property name="text">
         <string>Skip empty lines</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="SkipCommentLines_checkBox">
        <property name="text">
         <string>Skip comment lines</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="DeleteInputFile_checkBox">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Delete input file, be careful</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ClearAll_pushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>NumberOfLines_lineEdit_1</tabstop>
  <tabstop>NumberOfHeaderLines_lineEdit_1</tabstop>
  <tabstop>NumberOfHeaderLines_lineEdit_2</tabstop>
  <tabstop>NumberOfColumns_lineEdit_3</tabstop>
  <tabstop>NumberOfMetadataColumns_lineEdit_3</tabstop>
  <tabstop>SkipEmptyLines_checkBox</tabstop>
  <tabstop>SkipCommentLines_checkBox</tabstop>
  <tabstop>DeleteInputFile_checkBox</tabstop>
  <tabstop>ClearAll_pushButton</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
